ReleaseNotes.md

```diff
@@ -1,3 +1,103 @@
+- All vehicle states are now clearly in or out of the right of way, [no unknown state](https://github.com/openmobilityfoundation/mobility-data-specification/issues/770)
+- Many [new provider IDs added](https://github.com/openmobilityfoundation/mobility-data-specification/pulls?q=is%3Apr+is%3Aclosed+label%3A%22identifier+change%22+milestone%3A2.0.0)
+
+#### **_Modes Architecture_** 
+
+- [Support for multiple modes/services in MDS](https://github.com/openmobilityfoundation/mobility-data-specification/issues/574) 
+   - Updates the base of MDS to have some shared objects and fields, and specific fields as needed for each mode.
+   - Adds specific modes to MDS, with help from Member Networks, and existing real world data exchanges between operators, agencies, and solution providers.
+   - Operators must register a unique UUID for each mode they operate under. 
+
+**Passenger Services**
+- [Passenger services/TNC/taxi support ](https://github.com/openmobilityfoundation/mobility-data-specification/issues/95) 
+
+**Delivery Robots**
+- [Support for vehicles like delivery robots](https://github.com/openmobilityfoundation/mobility-data-specification/issues/782)  
+
+**Car Share**
+- [Carshare Support](https://github.com/openmobilityfoundation/mobility-data-specification/issues/640) 
+
+**Work to bring modes together**
+
+- [Add a "Data Provider UUID" to MDS](https://github.com/openmobilityfoundation/mobility-data-specification/issues/805) 
+   - New data provider id allows endpoints to include who is producing and serving up the data. Software companies and solution providers are [encouraged to register](https://github.com/openmobilityfoundation/mobility-data-specification/tree/feature-modes-cleanup#software-companies-using-mds) for their own global UUID now to serve up operator or agency data with MDS.
+- [Support for Modes in Policy](https://github.com/openmobilityfoundation/mobility-data-specification/issues/614) - specify which mode your policy applies to
+
+#### **_Policy Reimagining_**
+
+A reimagining of Policy, including top ten most common policies are clearly defined, edge cases (dwell time, trip definition, rule units, updating/ending policies, lookback periods) are clarified, Stops is out of beta, and Policy feeds are public
+
+- [Support utilization policies](https://github.com/openmobilityfoundation/mobility-data-specification/issues/620)
+- [Clarify minimum vehicle distribution](https://github.com/openmobilityfoundation/mobility-data-specification/issues/619)
+- [Support vehicle distribution by percent](https://github.com/openmobilityfoundation/mobility-data-specification/issues/618)
+- [Multimodal support in Policy](https://github.com/openmobilityfoundation/mobility-data-specification/issues/614)
+- [Clarification on dwell time duration](https://github.com/openmobilityfoundation/mobility-data-specification/issues/711)
```

Comment:
This issue is still open.

Reply:
You are right for issue #711, thank you. It's labeled for a future release. Removed this from the release notes.